user-api/src/main/kotlin/com/sns/user/component/user/repositories/UserRepository.kt

```diff
@@ -0,0 +1,8 @@
+package com.sns.user.component.user.repositories
+
+import com.sns.user.component.user.domains.User
+import org.springframework.data.repository.CrudRepository
+import org.springframework.data.repository.NoRepositoryBean
+
+@NoRepositoryBean
```

Comment:
https://docs.spring.io/spring-data/commons/docs/current/api/org/springframework/data/repository/NoRepositoryBean.html
```
Annotation to exclude repository interfaces from being picked up and thus in consequence getting an instance being created.

This will typically be used when providing an extended base interface for all repositories in combination with a custom repository base class to implement methods declared in that intermediate interface. In this case you typically derive your concrete repository interfaces from the intermediate one but don't want to create a Spring bean for the intermediate interface.
```

오 이렇게 하면 bean 주입에서 빠지나보네요 👍